### PR TITLE
Update app to use SDK version 1.0.4.

### PR DESCRIPTION
**Pull request by tmspzz**
- Update app to use SDK version 1.0.4. 
- Use Xcode default tab width.  (It's 4 spaces FYI)
- Fix Tests. (Test target was completely broken)

### CODE REVIEW

**Comment by tmspzz** [comment]
@mattmakai @philnash any interest in merging this?

File: twiliochat/MessagingManager.swift [hunk 1]
```diff
@@ -1,177 +1,189 @@
...
+        
+        TwilioChatClient.chatClient(withToken: token, properties: nil, delegate: self) { [weak self] result, chatClient in
+            
+            guard (result?.isSuccessful() ?? false) else { return }
+            
+            UIApplication.shared.isNetworkActivityIndicatorVisible = true
+            
+            self?.connected = true
+            self?.client = chatClient
+        }
```
Comment (well1791) [inline twiliochat/MessagingManager.swift]: Hi @blender 

Thanks for opening this PR, I really appreciate your effort but, It's hard to see the differences with all those indentation changes, would you mind to separate your changes in two commits? one for aesthetics (like indentation, code-style) and another one with the real code changes?

Also, I think every reviewer would appreciate clear commits.
Thanks in advance.

Comment (tmspzz) [reply twiliochat/MessagingManager.swift]: @well1791 there are now 3 separate commits

1) Adapt to new SDK
2) Fix tests
3) Fix Indentation

**Review by well1791 — APPROVED** [review]
This LGTM 👍 Thanks for opening this PR! Also, I'm going to close #10 and #19.

**Pull request merged**